pinot-plugins/pinot-batch-ingestion/v0_deprecated/pinot-spark/pom.xml

```diff
@@ -33,9 +33,9 @@
   <url>https://pinot.apache.org/</url>
   <properties>
     <pinot.root>${basedir}/../../../..</pinot.root>
-    <spark.version>2.4.0</spark.version>
-    <scala.binary.version>2.11</scala.binary.version>
-    <scala.version>2.11.11</scala.version>
+    <spark.version>2.4.7</spark.version>
+    <scala.binary.version>2.12</scala.binary.version>
```

Comment:
Do we know why we need to bump up the Scala version in this module? This module is in v0_deprecated module and it shouldn't affect the one in standalone module (correct me if I'm wrong). And directly bumping up Scala version may cause some runtime issue on some of the existing platforms which are running with Scala 2.11.

Reply:
FWIW, Flink recently dropped support for 2.11, partly because too many updated dependencies weren't built against it.